x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// TODO: Remove once typescript definitions are in EUI
+declare module '@elastic/eui' {
+  export const EuiBasicTable: React.SFC<any>;
+}
+
+import moment from 'moment';
+import React, { Component } from 'react';
+import chrome from 'ui/chrome';
+import { toastNotifications } from 'ui/notify';
+// @ts-ignore: implicit any for JS file
+import { Poller } from '../../../../common/poller';
+import { downloadReport } from '../lib/download_report';
+import { jobQueueClient } from '../lib/job_queue_client';
+import { ReportErrorButton } from './report_error_button';
+
+import {
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+
+interface Job {
+  id: string;
+  type: string;
+  object_type: string;
+  object_title: string;
+  created_by?: string;
+  created_at: string;
+  started_at?: string;
+  completed_at?: string;
+  status: string;
+  max_size_reached: boolean;
+}
+
+interface Props {
+  xpackInfo: any;
+  kbnUrl: any;
+}
+
+interface State {
+  page: number;
+  total: number;
+  jobs: Job[];
+  isLoading: boolean;
+}
+
+export class ReportListing extends Component<Props, State> {
+  private mounted?: boolean;
+  private poller?: any;
+  private isInitialJobsFetch: boolean;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      page: 0,
+      total: 0,
+      jobs: [],
+      isLoading: false,
+    };
+
+    this.isInitialJobsFetch = true;
+  }
+
+  public render() {
+    return (
+      <EuiPage className="repReportListing__page">
+        <EuiPageBody restrictWidth>
+          <EuiPageContent horizontalPosition="center">
+            <EuiTitle>
+              <h1>Reports</h1>
+            </EuiTitle>
+            {this.renderTable()}
+          </EuiPageContent>
+        </EuiPageBody>
+      </EuiPage>
+    );
+  }
+
+  public componentWillUnmount() {
+    this.mounted = false;
+    this.poller.stop();
+  }
+
+  public componentDidMount() {
+    this.mounted = true;
+    const { jobsRefresh } = chrome.getInjected('reportingPollConfig');
+    this.poller = new Poller({
+      functionToPoll: () => {
+        return this.fetchJobs();
+      },
+      pollFrequencyInMillis: jobsRefresh.interval,
+      trailing: false,
+      continuePollingOnError: true,
+      pollFrequencyErrorMultiplier: jobsRefresh.intervalErrorMultiplier,
+    });
+    this.poller.start();
+  }
+
+  private renderTable() {
+    const tableColumns = [
+      {
+        field: 'object_title',
+        name: 'Report',
+        render: (objectTitle: string, record: Job) => {
+          return (
+            <div>
+              <div>{objectTitle}</div>
+              <div>{record.object_type}</div>
```

Comment:
Thanks for fixing this page! One small comment.

Use something to distinguish these two lines apart. Your options are always the same in this kind of design for text situation.

1. Make one a different color.
2. Make one a different size.
3. Make one a different weight.
3. Combo of the three above.

I'd recommend making the second line subdued and a smaller size. That'll make the title (which is more important) stand out more. You could then bold the title if you need more oompf.